mne_bids/path.py

```diff
@@ -120,22 +128,28 @@ def __str__(self):
                     key = 'rec'
                 basename.append('%s-%s' % (key, val))
 
-        if self.suffix is not None:
-            basename.append(self.suffix)
+        if self.kind is not None:
+            if self.ext is not None:
+                basename.append(f'{self.kind}{self.ext}')
+            else:
+                basename.append(self.kind)
 
         basename = '_'.join(basename)
-        if self.prefix is not None:
-            basename = op.join(self.prefix, basename)
-
         return basename
 
+    def __str__(self):
+        """Return the string representation of the path."""
+        return self.basename
```

Comment:
mmh... why? :)

I'd prefer to get the full path here. If I want the basename … I will access `self.basename`.